spec/jsSpecs/ares-inspect.spec.js

```diff
@@ -98,7 +97,7 @@ describe(aresCmd, function() {
 
         setTimeout(() => {
             child.kill();
-            expect(result).not.toBeNull();
+            expect(result).toContain('Application Debugging - http://localhost');
```

Comment:
Was there a bug in this part? Why did you change it?
I think it's correct to check this when stdout is received.

if received error, this line is not meaningful.

Reply:
When the result is null, "toContain" would be failed. 

"toContain" can check the "result" is not null. right?